server/services/transactionMetadata.go

```diff
@@ -0,0 +1,47 @@
+package services
+
+import "github.com/ElrondNetwork/elrond-go-core/data/transaction"
+
+func extractTransactionMetadata(tx *transaction.ApiTransactionResult) objectsMap {
+	metadata := objectsMap{
+		"timestamp":         tx.Timestamp,
+		"value":             tx.Value,
+		"nonce":             tx.Nonce,
+		"typeOnSource":      tx.ProcessingTypeOnSource,
+		"typeOnDestination": tx.ProcessingTypeOnDestination,
+		"epoch":             tx.Epoch,
+		"sender":            tx.Sender,
+		"receiver":          tx.Receiver,
```

Comment:
if it is an esdt transfer, the receiver is not always in the tx.Receiver field

Reply:
Thanks! We will handle that one here: https://github.com/ElrondNetwork/rosetta/pull/43